Clockwise.Redis/RedisCircuitBreakerStorageSettings.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+
+namespace Clockwise.Redis
+{
+    public sealed class RedisCircuitBreakerStorageSettings
```

Comment:
Not sure we really need this class. Why not just move these constructor parameters to the `RedisCircuitBreakerStorage` constructor?